src/EditorFeatures/CSharp/ChangeSignature/CSharpChangeSignatureCommandHandler.cs

```diff
@@ -13,7 +14,8 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.ChangeSignature
     internal class CSharpChangeSignatureCommandHandler : AbstractChangeSignatureCommandHandler
     {
         [ImportingConstructor]
-        public CSharpChangeSignatureCommandHandler()
+        public CSharpChangeSignatureCommandHandler(IThreadingContext threadingContext) :
```

Comment:
minor - I believe @CyrusNajmabadi just changed a bunch of these to move the colon to the line with base.
Until there's an actual style setting for it though I don't mind.